azure/linux-webapp/outputs.tf

```diff
@@ -32,3 +32,8 @@ output "fqdn" {
   description = "Default FQDN of the app service."
   value       = azurerm_linux_web_app.main.default_hostname
 }
+
+output "slots" {
+  description = "Object containing details for the created slots."
```

Comment:
since there is a for_each on slots, would this be a list of objects? or perhaps an object of objects rather than just an object?

Reply:
It would be an object of objects in this case e.g.

resource.main would look like the below when for_each'd

```
{
  "first" = {
    "id" = "bla"
    "name" = "bla"
  }
  "second" = {
    "id" = "bla"
    "name" = "bla"
  }
}
```

Reply:
Thought so, should we describe it as such in the description field to make it easier on the module user?

Reply:
Whilst not technically needed, it does make it more verbose which would make it easier to understand yeah